### PR TITLE
Replaced .iteritems() with six.iteritems() for Python 2 and 3 compat

### PR DESCRIPTION
**Pull request by igormp**
Fixed some python 2 leftovers, as discussed in #267. 
Also formatted code according to Black, I hope that's not a problem.

This possibly breaks some python 2 compatibility.

### CODE REVIEW

**Comment by igormp** [comment]
As an addendum, I've noticed the usage of `six.iteritems()` in order to keep both python 2 and 3 compatibility. I wasn't sure if I should've kept using it instead of `.items()`, but I can't see why we should support Python 2 anyway.

**Comment by ganeshtata** [comment]
@igormp Thank you for the contribution. But isn't `.items()` supported in both Python2  and Python 3?

**Comment by igormp** [comment]
It is indeed, but might be way slower in Python 2 (at least according to [this](https://stackoverflow.com/a/10458567/4542015)).

**Comment by pietermarsman** [comment]
> but I can't see why we should support Python 2 anyway.

For now we support both python 2 and 3. Perhaps that will be changed in the future. Dropping python 2 is discussed (as you already know) in #194.

> Also formatted code according to Black, I hope that's not a problem.

Only format the lines you actually change. Formatting lines written by others messes up the git line history which can come in handy if you want to figure out why things are made in a particular way.

> As an addendum, I've noticed the usage of six.iteritems() in order to keep both python 2 and 3 compatibility.

As I see it there are 3 options: `.iteritems()`, `.items()`, and `six.iteritems()`. The does not exist in python 3, so we should not use that. There are still 8 occurances of `.iteritems()` and they should be replaced in this PR. `.items()` only works well in python 3, in python 2 it can cause memory issues because it returns a list instead of a generator, so we should not use it. I think `six.iteritems()` is the best solution because it works in both 2 and 3 and does not have memory issues. 

In short: I suggest we replace all occurances of `.iteritems()` and `.items()` with `six.iteritems()`.

**Comment by pietermarsman** [comment]
This is an (active) duplicate of #123.

File: pdfminer/cmapdb.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def use_cmap(self, cmap):
         assert isinstance(cmap, CMap), str(type(cmap))
 
         def copy(dst, src):
-            for (k, v) in src.iteritems():
+            for (k, v) in src.items():
```
Comment (pietermarsman) [inline pdfminer/cmapdb.py]: Use `six.iteritems()` instead.

File: pdfminer/pdfdevice.py [hunk 2]
```diff
@@ -9,14 +9,13 @@
...
 class PDFDevice(object):
-
     def __init__(self, rsrcmgr):
         self.rsrcmgr = rsrcmgr
         self.ctm = None
         return
 
     def __repr__(self):
-        return '<PDFDevice>'
+        return "<PDFDevice>"
```
Comment (pietermarsman) [inline pdfminer/pdfdevice.py]: Could you revert the reformatting of lines that are not changed intrinsically?

**Comment by igormp** [comment]
Ok, redid it. I can squash the commits after someone reviews the new changes.

I also think that it's a good idea to rename the PR since we're going to keep `six`.

**Comment by igormp** [comment]
Squashed the commits and changed the title, everything is good to go.
